include/common/logger.h

```diff
@@ -1 +1,55 @@
-class logger{};
\ No newline at end of file
+#pragma once
+#include <string>
+#include <fstream>
+#include <ctime>
+#include <chrono> 
+
+class Logger {
+public:
+	enum loglevel
+	{
+		logCRITICAL, logERROR, logWARNING, logDEBUG, logINFO,
+	};
+	Logger(std::string file_name, loglevel level) : full_buffer_size (100)
+	{
+		log_level_needed = level;
+		out.open(file_name, std::ofstream::out | std::ofstream::trunc);
```

Comment:
use std::ofstream::is_open to check that file was successfully opened, otherwise handle the error. simple assert() should be enough for now, till we don't have vfs implemented